tests/Database/DatabaseEloquentFactoryTest.php

```diff
@@ -182,23 +182,19 @@ public function test_after_creating_and_making_callbacks_are_called()
 
     public function test_has_many_relationship()
     {
-        $users = FactoryTestUserFactory::times(10)
-                        ->has(
-                            FactoryTestPostFactory::times(3)
-                                    ->state(function ($attributes, $user) {
-                                        // Test parent is passed to child state mutations...
-                                        $_SERVER['__test.post.state-user'] = $user;
-
-                                        return [];
-                                    })
-                                    // Test parents passed to callback...
-                                    ->afterCreating(function ($post, $user) {
-                                        $_SERVER['__test.post.creating-post'] = $post;
-                                        $_SERVER['__test.post.creating-user'] = $user;
-                                    }),
-                            'posts'
-                        )
-                        ->create();
+        $postFactory = FactoryTestPostFactory::times(3)
+            ->state(function ($attributes, $user) {
+                // Test parent is passed to child state mutations...
+                $_SERVER['__test.post.state-user'] = $user;
+
+                return [];
+            })
+            // Test parents passed to callback...
+            ->afterCreating(function ($post, $user) {
+                $_SERVER['__test.post.creating-post'] = $post;
+                $_SERVER['__test.post.creating-user'] = $user;
+            });
+        FactoryTestUserFactory::times(10)->has($postFactory, 'posts')->create();
```

Comment:
Revert the code style changes you made here and only focus on the PHPUnit fixes.